Fix lg-alert role attibute not being read by VoiceOver (#1385)
Legal-and-General/canopy: projects/canopy/src/lib/alert/docs/guide.stories.mdx

## Role

> It can be manually set through property binding or by passing the "none" value, preventing it from being set altogether. Otherwise, the role attribute is automatically assigned as 'alert' for any non-info and generic variants.

Reviewer: To me this currently reads as !info && generic, rather than !info && !generic.

```suggestion
It can be manually set through property binding or by passing the "none" value, preventing it from being set altogether. Otherwise, the role attribute is automatically assigned as 'alert' for anything other than `info` and `generic` variants.
```
